src/vcpkg/commands.cache.cpp

```diff
@@ -15,11 +15,10 @@ namespace vcpkg::Commands::Cache
         std::vector<BinaryParagraph> output;
         for (auto&& path : paths.get_filesystem().get_files_non_recursive(paths.packages(), VCPKG_LINE_INFO))
         {
-            const auto pghs = Paragraphs::get_single_paragraph(paths.get_filesystem(), path / "CONTROL");
-            if (const auto p = pghs.get())
+            auto pghs = Paragraphs::get_single_paragraph(paths.get_filesystem(), std::move(path) / "CONTROL");
+            if (auto p = pghs.get())
```

Comment:
```suggestion
            if (const auto p = pghs.get())
```

The pointer can be const.